src/client/power-pages/actions-hub/tree-items/ActiveGroupTreeItem.ts

```diff
@@ -36,7 +36,8 @@ export class ActiveGroupTreeItem extends ActionsHubTreeItem {
                 name: site.Name,
                 dataModelVersion: site.DataModel == WebsiteDataModel.Standard ? 1 : 2,
                 websiteUrl: site.WebsiteUrl,
-                status: WebsiteStatus.Active
+                status: WebsiteStatus.Active,
+                isCurrent: false //TODO: Implement this
```

Comment:
Comment is for identifying the current site based on active editor right?

Reply:
Yes, this has to be implemented based on active editor and workspace.